registry/storage/paths.go

```diff
@@ -69,38 +69,42 @@ const (
 //
 // We cover the path formats implemented by this path mapper below.
 //
-//	Manifests:
+//			Manifests:
```

Comment:
Should it just one tab?
```suggestion
//	Manifests:
```

Reply:
Changed with one tab.